lib/dao.ts

```diff
@@ -37,16 +38,49 @@ export class DAO {
     return dao;
   }
 
+  /**
+   * Returns promise of the DAOstack Genesis avatar address, or undefined if not found
```

Comment:
why return the promise, instead of the address itself? and why not return the DAO object?

Reply:
My reasoning is that sometimes people just want the address without the overhead of fetching the whole DAO.  They can call `DAO.at` if they want the DAO.